packages/next/src/build/turborepo-access-trace/types.ts

```diff
@@ -0,0 +1,30 @@
+// A single Addr / Port pair that was accessed during the duration of the trace
+export interface Address {
+  addr: string
+  port: string
+}
+
+// Tracked environment variable keys that were accessed during the duration of the trace
+export type EnvVars = Set<string | Symbol>
+// Tracks the file system paths that were accessed during the duration of the trace
+export type FS = Set<string>
+// Tracked Addr / Port pairs that were accessed during the duration of the trace
+export type Addresses = Array<Address>
+
+// The serializable version of TurborepoAccessTraceResult - this is required to pass the TurborepoAccessTraceResult
+// between workers where Sets are not serializable.
+export type SerializableTurborepoAccessTraceResult = Readonly<{
+  fs: Array<string>
+  addresses: Addresses
+  envVars: Array<string>
+}>
+
+// The public version of TurborepoAccessTraceResult - this is what is written to the trace file
```

Comment:
When defining comments on types or exported variables, you should use the JSDoc syntax to ensure that it can be read by hovering.

Reply:
Good suggestion, updated all of these and went back to clean up existing JSDoc strings.